Cargo.toml

```
@@ -13,6 +13,7 @@ description = "Fuel cryptographic primitives."
 [dependencies]
 borrown = "0.1"
 fuel-types = { version = "0.3", default-features = false }
+hex = { version = "0.4.3", default-features = false, features = ["std"] }
```

Unless there's a specific reason, prefer major.minor instead of major.minor.patch

```suggestion
hex = { version = "0.4", default-features = false, features = ["std"] }
```